app/models/user.rb

```diff
@@ -400,6 +409,8 @@ def phone_should_be_unique
       end
     end
 
+
+
```

Comment:
I think I might've moved the method from there to another part of the file and my linter didn't catch the extra blanks. I'll take care of it.